src/dmd/globals.d

```diff
@@ -159,6 +159,7 @@ struct Param
      * used to hide a feature that will have to go through deprecate-then-error
      * before becoming default.
      */
+    bool vmarkdown;         // list instances of Markdown replacements in Ddoc
```

Comment:
What's the `v` for?

Reply:
"verbose", I believe. I was copying the naming of existing fields like `vtls` and `vfield` which also trigger the listing of things.

Reply:
FWIW its `v` because it is associated with a flag e.g. `-vtls`. They exist to ease transition _to_ some new feature, by pointing out where the new behaviour is occurring (`-vtls`) or where old behaviour exists (e.g. `-vgc`). I'm not sure why a transition flag is necessary here.

Reply:
Thanks for the explanation of the `v`.

A transition flag was requested in [the previous incarnation of this PR](https://github.com/dlang/dmd/pull/8043#issuecomment-374395235) because supporting markdown styling changes the output of existing ddoc files that accidentally already have markdown syntax in them.

Reply:
That sounds reasonable then.

Reply:
Yes. Adding a global.parse.markdown boolean to enable the new markdown features will be a good idea.

Reply:
I have a couple questions about that:

1. I don't see an existing `global.parse` structure in globals.d; am I missing something?
2. How would that boolean get set? 

Currently we have `global.params.vmarkdown` (which is what we're discussing here), whose job is to print all instances of markdown replacements when the `transition=markdown` cli flag is provided. A second boolean might be confusing.

Reply:
I'm fine with adding a switch to enable markdown features. Since I'm a bit new to dmd development I'd appreciate some guidance:

1. Would the markdown features be enabled or disabled by default?
2. What would be a good name for the cli flag to use for the switch?

Reply:
> I'm fine with adding a switch to enable markdown features

Isn't that what `transition=markdown` is for?

Reply:
No, as the cli docs for `transition=markdown` describe, it's to "list instances of Markdown replacements in Ddoc." In other words, when `transition=markdown` is passed on the cli, every time we do a Markdown replacement we log it.

Reply:
I'd add:
```
transition=vmarkdown  // list instances
transition=markdown // enable new markdown features
```

Reply:
I understand, but if you look at `-transition` there are several others in the same situation.